Cargo.toml

```
@@ -14,8 +14,9 @@ homepage = "http://os.phil-opp.com/kernel-heap.html#a-better-allocator"
 [features]
 default = ["use_spin", "const_mut_refs"]
```

The `use_spin_nightly` feature should be enabled by default to make this change non-breaking:

```suggestion
default = ["use_spin_nightly", "const_mut_refs"]
```

---

I agree, done.